ui/components/app/detected-token/detected-token.test.js

```diff
@@ -28,6 +28,68 @@ describe('DetectedToken', () => {
             },
           },
         },
+        tokensChainsCache: {
+          [CHAIN_IDS.SEPOLIA]: {
```

Comment:
Hmm I thought we set this mock data in the storybook testData we are importing here?
It seems that this test is expecting Sepolia, whereas our test data uses Ethereum?

Reply:
Actually this is fine, it is a single case, we don't need to refactor mocks.